op-node/node/client.go

```diff
@@ -197,11 +197,15 @@ func (cfg *L1BeaconEndpointConfig) Setup(ctx context.Context, log log.Logger) (c
 		opts = append(opts, client.WithHeader(hdr))
 	}
 
-	a := client.NewBasicHTTPClient(cfg.BeaconAddr, log, opts...)
 	if cfg.BeaconArchiverAddr != "" {
-		b := client.NewBasicHTTPClient(cfg.BeaconArchiverAddr, log)
-		fb = append(fb, sources.NewBeaconHTTPClient(b))
+		addrs := strings.Split(cfg.BeaconArchiverAddr, ",")
```

Comment:
Should we make `BeaconArchiverAddr` a slice and do the splitting in the CLI parsing? Ideally the flag definition would change to be a `StringSliceFlag` rather than just `StringFlag`, and should update the description as well.

Reply:
`StringSliceFlags` work slightly differently than splitting

from https://github.com/urfave/cli/issues/1003
```
go run main.go --test=a,b,c,d
what you need is this
go run main.go --test a --test b --test c --test d
```

Reply:
Implemented these changes and took the opportunity to rename the flag (while retaining the old alias).

Reply:
@trianglesphere I also saw this, but then checked the behavior for env vars, and there it supports splitting comma-separated values.

Reply:
I tested it with this minimal example
```go
package main

import (
	"fmt"
	"log"
	"os"

	"github.com/urfave/cli/v2"
)

func main() {
	app := &cli.App{
		Flags: []cli.Flag{
			&cli.StringSliceFlag{
				Name:    "entry",
				EnvVars: []string{"ENTRIES"},
				Usage:   "multiple entries",
			},
		},
		Action: func(c *cli.Context) error {
			entries := c.StringSlice("entry")
			fmt.Println("Entries:", entries, len(entries))
			return nil
		},
	}

	err := app.Run(os.Args)
	if err != nil {
		log.Fatal(err)
	}
}
```